.travis.yml

```diff
@@ -1,3 +1,3 @@
 language: node_js
 node_js:
-  - "6"
+  - "14"
```

Comment:
We should be using v12 (LTS) and _optionally_ testing against v14 (Latest) https://nodejs.org/en